minio/minio.py

```diff
@@ -235,6 +225,97 @@ def __del__(self):
         if hasattr(self, "_http"):  # Only required for unit test run
             self._http.clear()
 
+    @staticmethod
+    def _get_part_info(object_size: int, part_size: int) -> tuple[int, int]:
+        """Compute part information for object and part size."""
+        def _calc_part_info(object_size: int, part_size: int):
+            """Compute part information for object and part size."""
+            def _validate_sizes():
+                """Validate object and part size."""
+                if part_size > 0:
+                    if part_size < MIN_PART_SIZE:
+                        raise ValueError(
+                            f"part size {part_size} is not supported; "
+                            f"minimum allowed 5MiB"
```

Comment:
```suggestion
                            f"minimum allowed {MIN_PART_SIZE}"
```

Reply:
No. Showing `MIN_PART_SIZE` is not human readable.